lib/date_time_parser.ex

```diff
@@ -142,23 +141,37 @@ defmodule DateTimeParser do
   If a time cannot be determined, then it will not be assumed by default. If you supply `true`, then
   `~T[00:00:00]` will be assumed. You can also supply your own time, and the found tokens will be
   merged with it.
+
+  * `:parsers` Default `#{inspect(DateTimeParser.Parser.available_parsers())}`.
+  The parsers to use when analyzing the string. When `Parser.Tokenizer`, the appropriate tokenizer will be
+  used depending on the function used.
```

Comment:
* document that order matters
* document that you can avoid Tokenizer and go straight to (for eg) US-formatted DateTimeUS or DateUS.